app/public/js/services/firebaseService.js

```diff
@@ -1,5 +1,9 @@
 angular.module('FirebaseService', []).service('firebaseService', function($rootScope) {
 
+  this.setUserPassword = function(newPassword) {
+    return $rootScope.user.updatePassword(newPassword);
```

Comment:
You're right, I'll try to use firebase.auth().currentUser. The reason why it belongs in FirebaseService is because it uses firebase functions such as all the functions of FirebaseService? :p